lib/chef/provider/aws_rds_instance.rb

```diff
@@ -0,0 +1,35 @@
+require 'chef/provisioning/aws_driver/aws_provider'
+
+class Chef::Provider::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSProvider
+  REQUIRED_OPTIONS = %w(db_instance_identifier allocated_storage engine
+                        db_instance_class master_username master_user_password)
+
+  OTHER_OPTIONS = %w(engine_version multi_az iops publicly_accessible db_name db_port)
+
+  def update_aws_object(instance)
+    Chef::Log.warn("aws_rds_instance does not support modifying a started instance")
```

Comment:
I'm fine with that - if you won't need updates we can merge this PR without support for it.  We can add it as soon as we need it.  Which we should do preferably before we have stuff in production being managed :)